### ISSUE TITLE
VisibleDeprecationWarning with numpy 1.11

### ISSUE THREAD

**Issue by samuelstjean** [nipy/nibabel#459]
~~~python
/home/samuel/.local/lib/python2.7/site-packages/nibabel/orientations.py:223: VisibleDeprecationWarning: non integer (and non boolean) array-likes will not be accepted as indices in the future
  undo_reorder = np.eye(p + 1)[list(ornt[:, 0]) + [p], :]
~~~

Nothing dramatic now, but good to know it would need fixing in future version. This started appearing on numpy 1.11 as far as I know.

**Comment by dewet22** [comment]
Already fixed by c6582ca on Mar 12, I'm just as eagerly waiting for a new release since right now I have to squelch these warnings by hand.

**Comment by samuelstjean** [comment]
Ah, silly me then, it's true I did not try to run the latest master as it just started appearing for me, so no need to have this open then.
